gradle/dependencies.gradle

```diff
@@ -9,9 +9,9 @@ apply from: "$rootDirectory/gradle/compose.gradle"
 
 def kotlinVersion
 if (useCompose.toBoolean()) {
-    kotlinVersion = '1.4.30'
+    kotlinVersion = '1.4.31'
```

Comment:
You may as well remove the if-statement?

Reply:
Seems like this version doesn't play well with Compose just yet

Reply:
Rolled back Compose's to 1.4.30.
As we discussed yesterday, let's keep if-statement for such cases.